internal/lm/nvml_test.go

```diff
@@ -132,7 +133,19 @@ func TestSharingLabeler(t *testing.T) {
 			},
 		},
 		{
-			descrition: "config with mps replicas",
+			descrition: "config with mps replicas no-mig-devices",
```

Comment:
This field name should be changed as a followup.
```suggestion
			description: "config with mps replicas no-mig-devices",
```

Reply:
Done as a commit on-top here.